scripts/zkai_attention_kv_stwo_median_timing_gate.py

```
@@ -0,0 +1,694 @@
+            }
+        )
+    return buffer.getvalue()
+
+
+def validate_output_path(raw_path: pathlib.Path) -> pathlib.Path:
+    if raw_path.exists() and raw_path.is_symlink():
+        raise StwoMedianTimingGateError(f"output path must not be a symlink: {raw_path}")
+    path = raw_path.resolve()
+    evidence_root = EVIDENCE_DIR.resolve()
+    if not path.parent.exists():
+        raise StwoMedianTimingGateError(f"output parent does not exist: {path.parent}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)